pkg/utils/common-preset.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 const (
-	DefaultVolumeMountPath    = "/dev/shm"
+	DefaultVolumeMountPath    = "/data"
```

Comment:
Why do you change this? The /dev/shm is still needed for distributed inference use case.